src/client/public/client.test.ts

```diff
@@ -131,15 +131,26 @@ describe('superface client', () => {
       const client = new SuperfaceClient();
 
       await expect(client.getProfile('does/not-exist')).rejects.toThrow(
-        'Profile "does/not-exist" is not installed. Please install it by running `superface install does/not-exist`.'
+        `Profile not installed: does/not-exist
```

Comment:
the full text in test is useless and maintenance nightmare, just put the shortest string that verifies it's the correct error